Makefile

```diff
@@ -1,6 +1,13 @@
-.PHONY: lint
+.PHONY: lint mut
 lint:
 	golangci-lint run -E whitespace -E wsl -E wastedassign -E unconvert -E tparallel -E thelper -E stylecheck -E prealloc \
 	-E predeclared -E nlreturn -E misspell -E makezero -E lll -E importas -E ifshort -E gosec -E  gofmt -E goconst \
 	-E forcetypeassert -E dogsled -E dupl -E errname -E errorlint -E nolintlint --timeout 2m
 
+install-deps:
+	go get github.com/JekaMas/go-mutesting/cmd/go-mutesting@v1.1.1
```

Comment:
Why using your fork?

Not necessary to `go get`, just `go install`?

Reply:
I forked it to add MSI threshold as a config parameter